cassandra-analytics-core-example/src/main/java/org/apache/cassandra/spark/example/ExampleStorageTransportExtension.java

```diff
@@ -145,17 +145,19 @@ private void refreshTokens()
 
     private StorageCredentialPair generateTokens(long tokenCount)
     {
-        return new StorageCredentialPair(new StorageCredentials("writeAccessKeyId-" + tokenCount,
+        return new StorageCredentialPair("writeRegion",
+                                         new StorageCredentials("writeAccessKeyId-" + tokenCount,
                                                                 "writeSecretKey-" + tokenCount,
                                                                 "writeSessionToken-" + tokenCount),
+                                         "readRegion",
                                          new StorageCredentials(
                                          "readAccessKeyId-" + tokenCount,
                                          "readSecretKey-" + tokenCount,
                                          "readSessionToken-" + tokenCount));
     }
 
     @Override
-    public void onStageSucceeded(String clusterId, long objectsCount, long rowsCount, long elapsedMillis)
```

Comment:
Object count and row count are still there. Reported from the `onAllObjectsPersisted` method, which is called right before STAGE_READY. 
The change only removes the duplication, so that the code is simplified a bit.